kroki/config.py

```diff
@@ -33,18 +33,18 @@ def pre_validation(self, config: MkDocsBaseConfig, key_name: str) -> None:
 
 class KrokiPluginConfig(MkDocsBaseConfig):
     ServerURL = config_options.URL(default=os.getenv("KROKI_SERVER_URL", "https://kroki.io"))
-    EnableBlockDiag = config_options.Type(bool, default=True)
-    EnableBpmn = config_options.Type(bool, default=True)
-    EnableExcalidraw = config_options.Type(bool, default=True)
-    EnableMermaid = config_options.Type(bool, default=True)
-    EnableDiagramsnet = config_options.Type(bool, default=False)
-    HttpMethod = config_options.Choice(choices=["GET", "POST"], default="GET")
-    UserAgent = config_options.Type(str, default=f"{__name__}/{version}")
-    FencePrefix = config_options.Type(str, default="kroki-")
-    FileTypes = config_options.Type(list, default=["svg"])
-    FileTypeOverrides = config_options.Type(dict, default={})
-    TagFormat = config_options.Choice(choices=["img", "object", "svg"], default="img")
-    FailFast = config_options.Type(bool, default=False)
+    EnableBlockDiag = config_options.Type(bool, default=os.getenv("KROKI_ENABLE_BLOCK_DIAG", "True").lower() in ("true", "1", "t"))
+    EnableBpmn = config_options.Type(bool, default=os.getenv("KROKI_ENABLE_BPMN", "True").lower() in ("true", "1", "t"))
+    EnableExcalidraw = config_options.Type(bool, default=os.getenv("KROKI_ENABLE_EXCALIDRAW", "True").lower() in ("true", "1", "t"))
+    EnableMermaid = config_options.Type(bool, default=os.getenv("KROKI_ENABLE_MERMAID", "True").lower() in ("true", "1", "t"))
+    EnableDiagramsnet = config_options.Type(bool, default=os.getenv("KROKI_ENABLE_DIAGRAMSNET", "False").lower() in ("true", "1", "t"))
+    HttpMethod = config_options.Choice(choices=["GET", "POST"], default=os.getenv("KROKI_HTTP_METHOD", "GET"))
+    UserAgent = config_options.Type(str, default=os.getenv("KROKI_USER_AGENT", f"{__name__}/{version}"))
+    FencePrefix = config_options.Type(str, default=os.getenv("KROKI_FENCE_PREFIX", "kroki-"))
+    FileTypes = config_options.Type(list, default=os.getenv("KROKI_FILE_TYPES", "svg").split(","))
+    FileTypeOverrides = config_options.Type(dict, default=eval(os.getenv("KROKI_FILE_TYPE_OVERRIDES", "{}")))
```

Comment:
This allows potential malicious strings to be evaluated.

Reply:
Would love to get a better understanding of the risks here. How does this expose us to any new risk compared to what is already there?

````
      ServerURL: !ENV [KROKI_SERVER_URL, 'https://kroki.io']
      FencePrefix:  !ENV [KROKI_FENCE_PREFIX, false]
      EnableBlockDiag: !ENV [KROKI_ENABLE_BLOCKDIAG, true]
```

Reply:
I could imagine that this could be a good entry point for certain CI/CD-attacks that fiddle with environment variables.
`eval` is discouraged to use in general -> [further reading](https://docs.astral.sh/ruff/rules/suspicious-eval-usage/) 

If the `!ENV` implementation is prone to this, does not matter, bc it is out of the plugin scope, and therefore others will hopefully fix this, if so.

Reply:
Ah, stupid me. Of course, we shouldn't use eval there. Updated